aries-site/src/examples/components/page/PageExample.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Card, Grid, Heading, Paragraph, Page, PageContent } from 'grommet';
+
+export const PageExample = () => (
+  <Page kind="narrow">
+    <PageContent>
+      <Heading margin="none" level={1} size="small">
```

Comment:
I think we should have some margin above the header

Reply:
I was holding off on polishing the examples until we kinda had some discussion around if these examples should be low-fi or high once I hear other people's thoughts I'll change these example